ios/browser/api/password/brave_password_api.mm

```diff
@@ -0,0 +1,472 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/ios/browser/api/password/brave_password_api.h"
+
+#include "base/bind.h"
+#include "base/strings/sys_string_conversions.h"
+
+#include "components/password_manager/core/browser/password_form.h"
+#include "components/password_manager/core/browser/password_form_digest.h"
+#include "components/password_manager/core/browser/password_store_consumer.h"
+#include "components/password_manager/core/browser/password_store_interface.h"
+
+#include "ios/web/public/thread/web_thread.h"
+#include "net/base/mac/url_conversions.h"
+#include "url/gurl.h"
+
+#include "brave/ios/browser/api/password/brave_password_observer.h"
+#include "brave/ios/browser/api/password/password_store_listener_ios.h"
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
+
+namespace brave {
+namespace ios {
+password_manager::PasswordForm::Scheme PasswordFormSchemeForPasswordFormDigest(
```

Comment:
Is the name here supposed to be `PasswordManagerSchemeFromPasswordFormScheme` ? (opposite of one below)

Reply:
Yeah I guess it makes more sense, this is a scheme conversion for the form digest but it is also a tongue twister honestly. 

Fixed at https://github.com/brave/brave-core/pull/12360/commits/f1ed2d6425c64f0f9a7d7075c7af10d5d38b04c5